packages/core/src/account/schema.ts

```diff
@@ -0,0 +1,58 @@
+import { Address as zAddress } from "abitype/zod";
+import type { Chain, Transport } from "viem";
+import z from "zod";
+import type { PublicErc4337Client, SupportedTransports } from "../client/types";
+import type { SmartAccountSigner } from "../signer/types";
+import { getChain } from "../utils/index.js";
+
+export const BaseSmartAccountParamsSchema = <
```

Comment:
nit: can we call this `createBaseSmartAccountParamsSchema` since it's a lambda? that makes it a bit clearer especially in the `z.infer`